docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -0,0 +1,628 @@
+scenario forecasts will incorporate baseline forecasts and the expected impact of 
+new screening technologies and vaccination. All forecasts will represent the Swiss 
+Re’s insured population from the weighted blend of Chinese provinces. Forecasts 
+run from year 2020 to 2040.
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+.. todo::
+
+ add more literature background
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+IHME will estimate the yearly number of cases of benign and invasive cervical 
+cancer detected under specific cervical cancer screening practices and the yearly 
+number of deaths from undetected cervical cancer (both in unit of per 100,000 
+insured person-years) in order to identify pay-out trends for cervical cancer 
+claims.
+
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal variables
+--------------------
+ 
+Outcome
+ - Cervical cancer detection (benign and invasive)
+ - Mortality and morbidity of cervical cancer
+Most proximal determinant/exposure
+ - Cervical cancer status
+ - Cervical cancer screening
+ - HPV infection
+ - HPV vaccination status
+Confounders
+ - age
+ - sex
+Effect modifiers
+ - N/A
+Mediators
+ - N/A
+
+
+.. _4.0:
+
+4.0 Intervention
+++++++++++++++++
+There is an urgent need to implement the evidenced-based interventions (e.g. HPV 
+vaccination, cervical cancer screening, management of detected disease) for eliminating cervical cancer as a public health problem, such action must be 
+strategic in nature. [WHO cervical cancer elimination strategy]_
+
+Based on SwissRe's interest, our simulation intervention combined the cervical 
+cancer screening and HPV vaccination to evaluate the extreme of the cervical 
+cancer detections in following scenarios:
+ - Baseline (lowest claim rate/payout): Scale-up of HPV vaccination while 
+   keeping cervical cancer screening constant over time among insured population.
+ - Alternative (highest claim rate/payout): Scale-up of cervical cancer 
+   screening while keeping HPV vaccination constant over time among insured population.
+
+.. _4.1:
+
+4.1 Simulation scenarios
+------------------------
+**Baseline:** by 2040, project existing level of cervical cancer screening for 
+women aged 21 to 65; linear ramp up HPV vaccination to cover 90% of the women 
+aged 15 to 45 in 2030, then stay 90% in 2031-2040. Where:
+ - Women without HPV infection are provided with cytology every three years
+ - Women with HPV infection are provided cytology every year
+
+**Alternative scenario:** by 2040, project existing level of HPV vaccination for 
+women aged 15 to 45; linear ramp up cervical cancer screening to cover 70% of the 
+women aged 21 to 65 in 2030, then stay 70% in 2031-2040. Where:
+ - Women without HPV infection are provided with cytology every three years
+ - Women with HPV infection are provided cytology every year
+
+.. image:: cervical_cancer_scale_up.PNG
+
```

Comment:
It looks like the dashed and solid lines for the HPV vaccine might be switched here in this figure

Reply:
Nevermind! It looks like I misunderstood initially